packages/ui/components/apps/Slider.tsx

```diff
@@ -48,13 +48,6 @@ export const Slider = <T extends string | unknown>({
 
   return (
     <div className={`mb-2 ${className}`}>
-      <style jsx global>
```

Comment:
this style is already added https://github.com/calcom/cal.com/blob/78656f2116192de5045855e13b0a49acc82a6e49/packages/ui/components/apps/Slider.tsx#L83